tests/rt.sh

```diff
@@ -240,7 +240,7 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   QUEUE=debug
   COMPILE_QUEUE=batch
 
-  #ACCNR=fv3-cpu
+  ACCNR=fv3-cpu
```

Comment:
This should be commented out again before the commit process

Reply:
Commented out.